### PR TITLE
change pull_request parameter in yamllint-check.yaml 

### PR DESCRIPTION
**Pull request by ShakedAviv50313**
change pull_request parameter in yamllint-check.yaml

### CODE REVIEW

File: .github/workflows/yamllint-check.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ on:
   pull_request:
     types:
       - opened
+      - edited
```
Comment (ernesgonzalez33) [inline .github/workflows/yamllint-check.yml:9]: @ShakedAviv50313 try deleting these three lines. Only with `pull_request` should be enough

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

File: tasks/dummy-dockerfile/dummy-dockerfile.yaml [hunk 2]
```diff
@@ -6,24 +6,26 @@ metadata:
...
+  workspaces:
     - name: dockerfilews
       description: The workspace where the dockerfile is stored
       mountPath: /tmp
   steps:
-    - name: create-dockerfile  
-      image: registry.access.redhat.com/ubi9/ubi-minimal:9.3-1361.1699548032
+    - name: create-dockerfile
+      image: |
+        registry.access.redhat.com/ubi9/ubi-minimal:9.3-1361.1699548032
```
Comment (ernesgonzalez33) [inline tasks/dummy-dockerfile/dummy-dockerfile.yaml:21]: You are adding a `\n` here that is making the pipeline fail

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

File: tasks/dummy-dockerfile/dummy-dockerfile.yaml (hunk 2 above)
Comment (ShakedAviv50313) [reply tasks/dummy-dockerfile/dummy-dockerfile.yaml:21]: i'm not sure how to change that, if i'm using it as a single-liner than it exceeds 80 chars and the yamllint fails

Comment (ernesgonzalez33) [reply tasks/dummy-dockerfile/dummy-dockerfile.yaml:21]: Try putting it in between quotes

File: tasks/dummy-dockerfile/dummy-dockerfile.yaml [hunk 3]
```diff
@@ -6,24 +6,25 @@ metadata:
...
+    - name: create-dockerfile
       image: registry.access.redhat.com/ubi9/ubi-minimal:9.3-1361.1699548032
       script: |
         #!/usr/bin/env sh
         set -xe
         if [ "$(workspaces.dockerfilews.bound)" == "true" ] ; then
-         echo -e "FROM alpine\nCMD ["echo","Hello World"]" > $(workspaces.dockerfilews.path)/$(params.subdirectory)/Dockerfile
-        else 
+         echo -e "FROM alpine\nCMD ["echo","Hello World"]" >
+         $(workspaces.dockerfilews.path)/$(params.subdirectory)/Dockerfile
```
Comment (ernesgonzalez33) [inline tasks/dummy-dockerfile/dummy-dockerfile.yaml:26]: @ShakedAviv50313 Better if you do it like this:
```
echo -e "FROM alpine\nCMD ["echo","Hello World"]" \
$(workspaces.dockerfilews.path)/$(params.subdirectory)/Dockerfile
```

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

**Pull request merged**